### ISSUE TITLE
The <pre> tag around the highlight div maybe shouldn't be added

### ISSUE THREAD

**Issue by wd** [niklasfasching/go-org#7]
According to this commit https://github.com/niklasfasching/go-org/commit/37d33c120656072472beba8bf3cf70e1d13ae14a , A `<pre>` was added around the `<div class="highlight">` , this cause the code block in my site looks ugly (check the comment for that commit ,there are some images). After some investigation, finally I found the reason why this happen.

Copy from https://www.w3schools.com/tags/tag_pre.asp
```
<pre>
Text in a pre element
is displayed in a fixed-width
font, and it preserves
both      spaces and
line breaks
</pre>
```
This is the html source generated by go-org, the line breaks will be preserved inside this tag, include the line break after every tags inside this block. So there will be 4 empty lines before and after the codes.

```
<pre class="src src-text"><div class="highlight"><div style="background-color:#fff;-moz-tab-size:4;-o-tab-size:4;tab-size:4">
<table style="border-spacing:0;padding:0;margin:0;border:0;width:auto;overflow:auto;display:block;"><tbody><tr><td style="vertical-align:top;padding:0;margin:0;border:0;">
<pre style="background-color:#fff;-moz-tab-size:4;-o-tab-size:4;tab-size:4"><code class="language-text" data-lang="text"><span style="margin-right:0.4em;padding:0 0.4em 0 0.4em;color:#7f7f7f">1
</span></code></pre></td>
<td style="vertical-align:top;padding:0;margin:0;border:0;">
<pre style="background-color:#fff;-moz-tab-size:4;-o-tab-size:4;tab-size:4"><code class="language-text" data-lang="text">$ celery -A proj beat</code></pre></td></tr></tbody></table>
</div>
</div>
</pre>
```

**Comment by niklasfasching** [comment]
Thx for catching this, I missed that. This change was introduced in an attempt to reproduce `org-export-html` behaviour more accurately but with your findings in mind I think this is misguided - we don't have as much control about how code is highlighted as `org-export-html`; We would have to tighten the requirements on `HighlightCodeBlock` and that's not too easy for our primary use case (hugo)...

TLDR: Let's use a wrapping `<div class="src src-xyz">` and leave the `<pre>` stuff to `HighlightCodeBlock`.

**Comment by niklasfasching** [comment]
closed in 97fe8b78
